app/scripts.babel/popup.js

```diff
@@ -143,14 +199,9 @@ limitations under the License.
         $('#connect-url').toggle();
       });
 
-      chrome.storage.sync.get('value', (res) => {
-        if (res.value && res.value.length > 0) {
-          $('#logo').css('background-image', 'url("../images/nuxeo-contrast.png")');
-          $('#connect-url-input').val(res.value);
-        } else {
-          $('#logo').css('background-image', 'url("../images/nuxeo_block-30.png")');
-        }
-      });
+      if (bkg.CONNECT_DOMAIN !== 'connect.nuxeo.com') {
```

Comment:
Is this working? Looks like `CONNECT_DOMAIN` is not "exported" as is `CONNECT_URL` [here](https://github.com/nuxeo/nuxeo-browser-extension/pull/14/files#diff-79f1810d2799a1f32a1de44d64b0d0f8R118)

Reply:
Yes it is